src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.TimeDate/Properties/Resources.resx

```diff
@@ -0,0 +1,235 @@
+    <value>Date</value>
+  </data>
+  <data name="Microsoft_plugin_timedate_DateAndTime" xml:space="preserve">
+    <value>Date and time</value>
+  </data>
+  <data name="Microsoft_plugin_timedate_DateAndTimeUtc" xml:space="preserve">
+    <value>Date and time UTC</value>
+    <comment>'UTC' means here 'Universal Time Convention'</comment>
+  </data>
+  <data name="Microsoft_plugin_timedate_Day" xml:space="preserve">
+    <value>Day (Week day)</value>
+  </data>
+  <data name="Microsoft_plugin_timedate_DayOfMonth" xml:space="preserve">
+    <value>Day of the month</value>
+  </data>
+  <data name="Microsoft_plugin_timedate_DayOfWeek" xml:space="preserve">
+    <value>Day of the week (Week day)</value>
+  </data>
+  <data name="Microsoft_plugin_timedate_DayOfYear" xml:space="preserve">
+    <value>Day of the year</value>
+  </data>
+  <data name="Microsoft_plugin_timedate_ErrorResultSubTitle" xml:space="preserve">
+    <value>Valid prefixes: 'u' for Unix Timestamp, 'ft' for Windows file time</value>
+  </data>
+  <data name="Microsoft_plugin_timedate_ErrorResultTitle" xml:space="preserve">
+    <value>Error: Invalid number input</value>
+  </data>
+  <data name="Microsoft_plugin_timedate_Iso8601" xml:space="preserve">
+    <value>ISO 8601 (Date and time)</value>
+  </data>
+  <data name="Microsoft_plugin_timedate_Iso8601Utc" xml:space="preserve">
+    <value>ISO 8601 UTC (Date and time)</value>
+    <comment>'UTC' means here 'Universal Time Convention'</comment>
+  </data>
+  <data name="Microsoft_plugin_timedate_Iso8601Zone" xml:space="preserve">
+    <value>ISO 8601 with time zone (Date and time)</value>
+  </data>
+  <data name="Microsoft_plugin_timedate_Iso8601ZoneUtc" xml:space="preserve">
+    <value>ISO 8601 UTC with time zone (Date and time)</value>
+    <comment>'UTC' means here 'Universal Time Convention'</comment>
+  </data>
+  <data name="Microsoft_plugin_timedate_Month" xml:space="preserve">
+    <value>Month</value>
+  </data>
+  <data name="Microsoft_plugin_timedate_MonthOfYear" xml:space="preserve">
+    <value>Month of the year</value>
+  </data>
+  <data name="Microsoft_plugin_timedate_Now" xml:space="preserve">
+    <value>Now (Current date and time)</value>
+  </data>
+  <data name="Microsoft_plugin_timedate_NowUtc" xml:space="preserve">
+    <value>Now UTC (Current date and time)</value>
+    <comment>'UTC' means here 'Universal Time Convention'</comment>
+  </data>
+  <data name="Microsoft_plugin_timedate_plugin_description" xml:space="preserve">
+    <value>Provides time and date values for the system time or a custom time stamp (Examples: 'day', 'day::2/3/2022', 'time::10:30:45 AM', 'calendar week::2/3/2022')</value>
+    <comment>The character sequence '::' is a fixed delimiter in plugin code. '2/3/2022' and '10:30:45 AM' are date/time values that should be globalized (Example for Germany: '3/2/2022' -&gt; '02.03.2022', '10:30:45 AM' -&gt; '10:30:45').</comment>
```

Comment:
Replace time/date examples with a place holder and fill them at runtime. The format has to be always in the system culture and therefore a fixed, translated example is a bad idea.